contracts/test/token/oeth.fork-test.js

```diff
@@ -0,0 +1,41 @@
+const { expect } = require("chai");
+
+const { loadDefaultFixture } = require("./../_fixture");
+const { isCI } = require("./../helpers");
+
+/**
+ * Regarding hardcoded addresses:
+ * The addresses are hardcoded in the test files (instead of
+ * using them from addresses.js) intentionally. While importing and
+ * using the variables from that file increases readability, it may
+ * result in it being a single point of failure. Anyone can update
+ * the addresses.js file and it may go unnoticed.
+ *
+ * Points against this: The on-chain data would still be correct,
+ * making the tests to fail in case only addresses.js is updated.
+ *
+ * Still open to discussion.
+ */
+
+describe("ForkTest: OETH", function () {
+  this.timeout(0);
+
+  // Retry up to 3 times on CI
+  this.retries(isCI ? 3 : 0);
+
+  let fixture;
+  beforeEach(async () => {
+    fixture = await loadDefaultFixture();
+  });
+
+  describe("verify state", () => {
+    // These tests use a transaction to call a view function so the gas usage can be reported.
+    it("Should get total value", async () => {
+      const { oeth } = fixture;
+      const eigenLayerStrategyContract =
+        "0xa4c637e0f704745d182e4d38cab7e7485321d059";
```

Comment:
nit: I'd add EigenLayerStrategy to the addresses file

Reply:
This is almost the same comment I've left on one of @shahthepro's PRs but he made a great point, that when these addresses are immutable, it is safer to leave them hardcoded in the test files, so someone doesn't mistakingly change the address in the `addresses.js` and that could have unexpected consequences. [This comment](https://github.com/OriginProtocol/origin-dollar/pull/1961/files#diff-0253f024335a35ec3acd15baf09dd345175408bd79d82eda7a5ed4883c3a82aaR6-R18) addresses it.

I think Shah makes a great point, though I don't have strong conviction either way regarding this topic.